user-guide/Standard_Apps/PA/Getting_Started_with_PA/Creating_and_Configuring_a_Process_Definition.md

```diff
@@ -0,0 +1,116 @@
+   - *PA DOM Reference*: Leave this set to *Not configured*.
+
+     ![Profile_Instance](~/user-guide/images/Profile_Instance.png)
+
+   Alternatively, start the process with a *PA Timer Start Event* that can be used to generate new tokens periodically (see [Pushing tokens periodically](xref:Pushing_Tokens_Into_A_Process#pushing-tokens-periodically)).
+
+1. In the *Process Automation* app, select the newly created process definition.
+
+1. Click the *Configure* button to open the *process definition configuration* wizard.
+
+1. On the first page of the wizard, select the process DOM definition and click *Next*. A page will be shown where all nodes in the process definition are listed.
+
+1. For each node in the process definition:
+
+   1. Select the node and click *Configure*.
+
+   1. Provide the requested information. This will mainly be related to the profile instance selection, profile instance creation, and the link with DOM field descriptors.
+
+      ![Process_Definition_Configuration](~/user-guide/images/Process_Definition_Configuration.png)
+
+      For example, for a "Ping IP" script task, the following input would be needed in the wizard:
+
+      - **Profile Definition**: The profile definition set up during the creation of the "Ping IP" script task.
+
+      - **Profile Instance**: An existing or new profile instance.
+
+      - **Profile Parameters**: A list of profile parameters that are part of the profile definition.
+
+        Parameters are used to define the input and output of the activity. Input parameters can contain hard-coded values. In that case, the same value will always be used. Alternatively, they can refer to a field descriptor from the process DOM instances. In that case, different values need to be used for each process run.
+
+        - Hard-coded value:
+
+          ![Hard-coded Value](~/user-guide/images/Hard_Coded_Value.png)
+
+        - Dynamic value:
+
+          ![Dynamic Value](~/user-guide/images/Dynamic_Value.png)
+
+        Output parameters should be linked to the field descriptor of the process DOM instance.
+
+1. Click the *Confirm* button.
+
+## Creating an activation window
+
+1. In the *Process Automation* app, select the newly created process definition.
+
+1. Click the *Activate* button.
+
+1. On the first page of the *create activation window* wizard, specify the following information:
+
+   - **Process name**: Specify a name for the activation window.
+
+   - **Service info**: This identifies the process and makes it possible to push tokens into the process from custom code. To configure this, there are two options:
+
+     - Select *Create New* to create *Service info* with the same name as the activation window.
+
+     - Select an existing *Service info* item in case multiple activation windows need to be created that should always be linked to the same process identifier.
+
+     ![Activation Window](~/user-guide/images/Activation_Window.png)
```

Comment:
I see in this screenshot that you can configure more info than the process name and service info in this wizard. Could you check this and add any info that is missing?